src/server.h

```diff
@@ -801,14 +801,14 @@ typedef struct RedisModuleIO {
 /* Macro to initialize an IO context. Note that the 'ver' field is populated
  * inside rdb.c according to the version of the value to load. */
 #define moduleInitIOContext(iovar,mtype,rioptr,keyptr,db) do { \
-    iovar.rio = rioptr; \
-    iovar.type = mtype; \
-    iovar.bytes = 0; \
-    iovar.error = 0; \
-    iovar.ver = 0; \
-    iovar.key = keyptr; \
-    iovar.dbid = db; \
-    iovar.ctx = NULL; \
+    (iovar).rio = (rioptr); \
+    (iovar).type = (mtype); \
+    (iovar).bytes = 0; \
+    (iovar).error = 0; \
+    (iovar).ver = 0; \
+    (iovar).key = (keyptr); \
+    (iovar).dbid = (db); \
+    (iovar).ctx = NULL; \
```

Comment:
since this macro and the two after it, are used specifically to initialized a variable, i think it's ok to skip the parenthesis around the variable.
and since the value on the right side is without any formula and followed by a semi-colon, then there's no risk of operator order being messed up.
i think we can revert them.